### PR TITLE
Adding small and medium scale control plane workloads for azure-aro

### PR DESCRIPTION
**Pull request by vishnuchalla**

### CODE REVIEW

**Comment by vishnuchalla** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aro-4.13-nightly-ocp-qe-perfscale-ci-tests-aro-node-density-heavy-small-scale

**Comment by vishnuchalla** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aro-4.13-nightly-ocp-qe-perfscale-ci-tests-aro-node-density-heavy-medium-scale

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aro-4.13-nightly.yaml [hunk 1]
```diff
@@ -40,6 +40,66 @@ tests:
     test:
     - chain: openshift-qe-cluster-density-v2
     workflow: openshift-qe-installer-azure-aro
+- as: ocp-qe-perfscale-ci-tests-aro-node-density-small-scale
```
Comment (krishvoor) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aro-4.13-nightly.yaml:43]: Please update the job names to `ocp-qe-perfscale-ci-tests-node-density-24nodes-aro`

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aro-4.13-nightly.yaml [hunk 2]
```diff
@@ -40,6 +40,66 @@ tests:
...
+  cron: 0 12 * * 2
+  steps:
+    cluster_profile: azure-qe
+    env:
+      ARO_WORKER_COUNT: "24"
+      ARO_WORKER_VM_SIZE: Standard_D8s_v4
+    test:
+    - chain: openshift-qe-node-density
+    workflow: openshift-qe-installer-azure-aro
+- as: ocp-qe-perfscale-ci-tests-aro-node-density-cni-small-scale
```
Comment (krishvoor) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aro-4.13-nightly.yaml:53]: Same comment as above

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aro-4.13-nightly.yaml [hunk 3]
```diff
@@ -40,6 +40,66 @@ tests:
...
+  cron: 0 12 * * 3
+  steps:
+    cluster_profile: azure-qe
+    env:
+      ARO_WORKER_COUNT: "24"
+      ARO_WORKER_VM_SIZE: Standard_D8s_v4
+    test:
+    - chain: openshift-qe-node-density-cni
+    workflow: openshift-qe-installer-azure-aro
+- as: ocp-qe-perfscale-ci-tests-aro-node-density-heavy-small-scale
```
Comment (krishvoor) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aro-4.13-nightly.yaml:63]: Same comment as above

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aro-4.13-nightly.yaml [hunk 4]
```diff
@@ -40,6 +40,66 @@ tests:
...
+  cron: 0 12 * * 4
+  steps:
+    cluster_profile: azure-qe
+    env:
+      ARO_WORKER_COUNT: "24"
+      ARO_WORKER_VM_SIZE: Standard_D8s_v4
+    test:
+    - chain: openshift-qe-node-density-heavy
+    workflow: openshift-qe-installer-azure-aro
+- as: ocp-qe-perfscale-ci-tests-aro-node-density-medium-scale
```
Comment (krishvoor) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aro-4.13-nightly.yaml:73]: Same comment as above

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aro-4.13-nightly.yaml [hunk 5]
```diff
@@ -40,6 +40,66 @@ tests:
...
+  cron: 0 12 * * 2
+  steps:
+    cluster_profile: azure-qe
+    env:
+      ARO_WORKER_COUNT: "120"
+      ARO_WORKER_VM_SIZE: Standard_D8s_v4
+    test:
+    - chain: openshift-qe-node-density
+    workflow: openshift-qe-installer-azure-aro
+- as: ocp-qe-perfscale-ci-tests-aro-node-density-cni-medium-scale
```
Comment (krishvoor) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aro-4.13-nightly.yaml:83]: Same comment as above

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aro-4.13-nightly.yaml [hunk 6]
```diff
@@ -40,6 +40,66 @@ tests:
...
+  cron: 0 12 * * 3
+  steps:
+    cluster_profile: azure-qe
+    env:
+      ARO_WORKER_COUNT: "120"
+      ARO_WORKER_VM_SIZE: Standard_D8s_v4
+    test:
+    - chain: openshift-qe-node-density-cni
+    workflow: openshift-qe-installer-azure-aro
+- as: ocp-qe-perfscale-ci-tests-aro-node-density-heavy-medium-scale
```
Comment (krishvoor) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aro-4.13-nightly.yaml:93]: Same comment as above

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aro-4.13-nightly.yaml (hunk 1 above)
Comment (krishvoor) [reply ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aro-4.13-nightly.yaml:43]: You can combine `node-density` & `cluster-density-v2` two jobs into 1  with this [chain](https://steps.ci.openshift.org/chain/openshift-qe-ocp-qe-perfscale-ci-tests)

**Review by krishvoor — CHANGES_REQUESTED** [review]

**Comment by vishnuchalla** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aro-4.13-nightly-ocp-qe-perfscale-ci-tests-aro-node-density-heavy-medium-scale

**Comment by krishvoor** [comment]
@vishnuchalla Can you consider running a small-scale load for now?

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aro-4.13-nightly.yaml (hunk 1 above)
Comment (vishnuchalla) [reply ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aro-4.13-nightly.yaml:43]: Done

**Comment by vishnuchalla** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aro-4.13-nightly-ocp-qe-perfscale-ci-tests-24nodes-aro

**Comment by vishnuchalla** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aro-4.13-nightly-ocp-qe-perfscale-ci-tests-120nodes-aro

**Comment by vishnuchalla** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aro-4.13-nightly-ocp-qe-perfscale-ci-tests-120nodes-aro

**Comment by vishnuchalla** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aro-4.13-nightly-ocp-qe-perfscale-ci-tests-24nodes-aro

**Comment by vishnuchalla** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aro-4.13-nightly-ocp-qe-perfscale-ci-tests-24nodes-aro

**Comment by vishnuchalla** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aro-4.13-nightly-ocp-qe-perfscale-ci-tests-node-density-cni-24nodes-aro

**Comment by vishnuchalla** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aro-4.13-nightly-ocp-qe-perfscale-ci-tests-node-density-cni-24nodes-aro

**Comment by vishnuchalla** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aro-4.13-nightly-ocp-qe-perfscale-ci-tests-node-density-cni-24nodes-aro

File: ci-operator/step-registry/openshift-qe/node-density-cni/openshift-qe-node-density-cni-chain.yaml [hunk 7]
```diff
@@ -0,0 +1,11 @@
+
+chain:
+  as: openshift-qe-node-density-cni
+  steps:
+  - ref: openshift-qe-node-density-cni
+  - ref: openshift-qe-write-perfscale-results
+  env:
+  - name: OUTPUT_FILE
+    default: "kube-burner.out"
```
Comment (paigerube14) [inline ci-operator/step-registry/openshift-qe/node-density-cni/openshift-qe-node-density-cni-chain.yaml:9]: This needs to be **kube-burner-ocp.out** to properly process the output file using the sandman script. Currently the sandman script in _openshift-qe-write-perfscale-results_ is set up for both kube-burner and kube-burner-ocp ways of running. To properly set the regex styling of output need to have kube-burner-ocp.out file name

**Comment by vishnuchalla** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aro-4.13-nightly-ocp-qe-perfscale-ci-tests-node-density-cni-24nodes-aro

**Comment by vishnuchalla** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aro-4.13-nightly-ocp-qe-perfscale-ci-tests-node-density-heavy-24nodes-aro

**Pull request closed without merging**